### ISSUE TITLE
How to make a gif loop

### ISSUE THREAD

**Issue by franleplant** [PistonDevelopers/image-gif#8]
Hi!
    I was able to create a gif successfully, but the sequence of frames runs only once and don't start over, or, which is the same, the gif does not loop.

How can I achieve that?

Thanks!

**Comment by nwin** [comment]
This is currently not possible. Please note that this is an issue tracker and not a help forum.

Closing in favor of #9.

**Comment by franleplant** [comment]
Ok, thanks!

**Comment by nwin** [comment]
@franleplant The API and the examples have been updated. It’s not published on crates.io yet.

**Comment by franleplant** [comment]
Cool, great job! Thanks a lot!
